### PR TITLE
Add ySOL to Strict List

### PR DESCRIPTION
**Pull request by 0xcounter**
Add Synatra Staked SOL to Strict List

# Validate [ySOL](https://solscan.io/token/yso11zxLbHA3wBJ9HAtVu6wnesqz9A2qxnhxanasZ4N)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/synatraxyz/status/1735208464324993379
- Coingecko/ CMC URL (If available): [ySOL](https://www.coingecko.com/en/coins/synatra-staked-sol)

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by zykalo** [comment]
check protocol good protocol
would be helpful now with the meteora pool that's been set up

**Comment by 0xJellyy** [comment]
A very solid platform to stake your SOL.

It's definitely worth putting ySOL on the Strict List.

**Comment by nikpopo** [comment]
good and currently unique project which need to have max support at the start of this beautiful DEFI journey

**Comment by MangoMatcha08** [comment]
I support adding ySol to the Jupiter token list. Been a year and they have had a clean track record.

**Comment by h8AjVHek** [comment]
Will be useful to have ySOL on the Strict List, Synatra is impressive.

**Comment by opptylabs** [comment]
Useful protocol

**Comment by Timoon21** [comment]
Hey @0xcounter Can you add your token at the end of the file? Not at beginning. Thx.

**Comment by Sushi-Roll-17** [comment]
Great protocol, Happy to see this becoming recognized!

**Comment by BlewEnzo** [comment]
Incredible, ground breaking protocol. Safe, secure, and excellent returns. The only place I stake my SOL for rewrads.

File: validated-tokens.csv [hunk 1]
```diff
@@ -623,6 +623,7 @@ Random Zaysan Raptors,rZOOM,Vjq9T5xmqRzLXQRyvigzyZzpHCGCsbYAJ7afLVuF8j9,9,https:
 Restack Coin,RESTACK,48ntzCLdm9apLbZ8JV5b2s3teMZ1ij1XqokJiqiRYmb3,9,https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/48ntzCLdm9apLbZ8JV5b2s3teMZ1ij1XqokJiqiRYmb3/logo.png,false
 Extra Reality,XTR,6ABQdaTwRvmacto7aeRBGghS6Pxctd6cFGL8gDdwV1dd,8,https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/6ABQdaTwRvmacto7aeRBGghS6Pxctd6cFGL8gDdwV1dd/logo.png,false
 Famous Fox Federation Solvent Droplet,svtFFF,BoeDfSFRyaeuaLP97dhxkHnsn7hhhes3w3X8GgQj5obK,8,https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/BoeDfSFRyaeuaLP97dhxkHnsn7hhhes3w3X8GgQj5obK/logo.png,false
+Synatra Staked SOL,ySOL,yso11zxLbHA3wBJ9HAtVu6wnesqz9A2qxnhxanasZ4N,9,https://shdw-drive.genesysgo.net/76HCWYyF9Sps1TnRdZB1CtHbj5Ji7trzqi4tnxfCabXE/ysol-logo.png,false
```
Comment (Timoon21) [inline validated-tokens.csv:626]:
```suggestion
Synatra Staked SOL,ySOL,yso11zxLbHA3wBJ9HAtVu6wnesqz9A2qxnhxanasZ4N,9,https://shdw-drive.genesysgo.net/76HCWYyF9Sps1TnRdZB1CtHbj5Ji7trzqi4tnxfCabXE/ysol-logo.png,true
```

**Review by Timoon21 — APPROVED** [review]

**Review by 9yoi — APPROVED** [review]

**Pull request merged**